tools/loader/ImageClassifier.cpp

```diff
@@ -101,6 +112,24 @@ static bool getNextImageFilenames(std::vector<std::string> *filenames) {
   return !filenames->empty();
 }
 
+/// Generate in \p imageList the list of filenames corresponding to the next
+/// mini-batch of size \p miniBatchSize extracted from \p totalImageList at
+/// index \p minibatchIndex.
+static bool getNextMiniBatch(std::vector<std::string> &imageList,
+                             std::vector<std::string> &totalImageList,
```

Comment:
nit: could use `llvm::ArrayRef<std::string> totalImageList` here. Or at least make this `const`.

Reply:
Done